addons/account/models/chart_template.py

```diff
@@ -269,7 +269,7 @@ def tax_template_changed(tax, template):
                         if (
                             command not in (Command.UPDATE, Command.CREATE)
                             or not self.ref(vals['tax_src_id'], raise_if_not_found=False)
-                            or not self.ref(vals['tax_dest_id'], raise_if_not_found=False)
+                            or not self.ref(vals.get('tax_dest_id'), raise_if_not_found=False)
```

Comment:
You only want to update the fiscal positions if there are new taxes linked to it though.
If there is no `tax_dest_id`, you can't consider it to be a new tax when you can't find it.